edge_computing/day_2_core_cnf_clusters/telco-day-2-welcome.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="telco-day-2-welcome"]
+= Day 2 operations for telco core CNF clusters
+include::_attributes/common-attributes.adoc[]
+:context: telco-day-2-welcome
+
+toc::[]
+
+What follows is an overview of Day 2 operations you can use to manage telco core CNF clusters.
+
+Upgrading a telco core CNF cluster:: Updating your cluster is a critical task that ensures that bugs and potential security vulnerabilities are patched. 
+For more information, see xref:../day_2_core_cnf_clusters/updating/telco-update-welcome.adoc#telco-update-welcome[Upgrading a telco core CNF cluster].
```

Comment:
I know core OCP has settled on update instead of upgrade. This uses both. Should these all be "update"?
```suggestion
Updating a telco core CNF cluster:: Updating your cluster is a critical task that ensures that bugs and potential security vulnerabilities are patched. 
For more information, see xref:../day_2_core_cnf_clusters/updating/telco-update-welcome.adoc#telco-update-welcome[Updating a telco core CNF cluster].
```
For the link, you'd also want to change the title in that module.

Reply:
yeah there was a mixture in the target docs also. I've gone with _Update_
Thanks for spotting that